auth-web/src/components/auth/account-deactivate/DeactivateCard.vue

```diff
@@ -21,35 +21,34 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop } from 'vue-property-decorator'
+import { computed, defineComponent, ref } from '@vue/composition-api'
 import Vue from 'vue'
+import { useI18n } from 'vue-i18n-bridge'
 
-@Component({
-})
-export default class DeactivateCard extends Vue {
-  @Prop() private type: string
-
-  // no type matches means show it for all types
-  private infoArray: { title: string, description?: string, type?: string }[] = [
-    {
-      title: 'deactivateMemberRemovalTitle',
-      description: 'deactivateMemberRemovalDesc'
-    },
-    {
-      title: 'businessRemovalTitle',
-      description: 'businessRemovalDesc'
-    },
-    {
-      title: 'padRemovalTitle',
-      type: 'PREMIUM'
-    }
+export default defineComponent({
+  name: 'DeactivateCard',
+  props: {
+    type: String
+  },
+  setup (props) {
+    const { t } = useI18n()
+    const infoArray = ref([
+      { title: 'deactivateMemberRemovalTitle', description: 'deactivateMemberRemovalDesc' },
+      { title: 'businessRemovalTitle', description: 'businessRemovalDesc' },
+      { title: 'padRemovalTitle', type: 'PREMIUM' }
+    ] as any)
 
-  ]
+    const info = computed(() => {
+      return infoArray.value.filter(obj => !obj.type || obj.type === props.type)
+    })
 
-  private get info () {
-    return this.infoArray.filter(obj => !obj.type || obj.type === this.type)
+    return {
+      infoArray,
+      t,
```

Comment:
you're returning `t` here, but you're using `$t` in the html. You'll need to change that if you want anything to happen. Did you test this locally? The way you have it currently looks like it would break the description

Reply:
updated the change in the template